### PR TITLE
Core: remove custom ready event

### PR DESCRIPTION
**Pull request by timmywil**
Fixes gh-2264

### CODE REVIEW

File: test/unit/ready.js [hunk 1]
```diff
@@ -1,73 +1,70 @@
-module("event");
+module( "event" );
 
-(function(){
+(function() {
```
Comment (markelog) [inline test/unit/ready.js]: Generally, it was asked to not fix code-style issues if PR is not about a code-style

**Comment by markelog** [comment]
io failures are related to https://github.com/jquery/jquery/issues/2266?

**Comment by timmywil** [comment]
Yup

File: test/unit/ready.js (hunk 1 above)
Comment (timmywil) [inline test/unit/ready.js]: Seems to me that the best opportunity to change code styles is while we're already in that code, but I'm not overly adamant.

**Comment by markelog** [comment]
LGTM

**Pull request closed without merging**